safe_gpu/safe_gpu.py

```diff
@@ -44,12 +44,22 @@ def pytorch_placeholder(device_no):
     import torch
     return torch.zeros((1), device=f'cuda:{device_no}')
 
-
 def tensorflow_placeholder(device_no):
     import tensorflow as tf
     with tf.device(f'GPU:{device_no}'):
         return tf.constant([1.0])
 
+def pycuda_placeholder(device_no):
+    # https://documen.tician.de/pycuda/driver.html#pycuda.driver.Device.make_context
+    print(f"pytorch_placeholder(device_no={device_no})")
```

Comment:
No printing here please. This print in particular looks like a developement artifact, I'd get rid of it altogether.